pkg/applicationserver/io/mqtt/mqtt.go

```diff
@@ -204,11 +195,6 @@ func (c *connection) setup(ctx context.Context) error {
 				err = c.mqtt.Send(pkt)
 			}
 			if err != nil {
-				if err != stdio.EOF {
```

Comment:
Since `c.mqtt` may be closed from a separate goroutine, this may actually log an error simply because the context closed. As such, this type of logging has been moved to the single point of closure - the goroutine on line 219.